backend-truffle/contracts/DoorEvent.sol

```diff
@@ -1,103 +1,58 @@
-//Meet.inc >>> ĐOor-Event <<<
-//Đoor-Event - Everything's decentralized but not your Đoor
-//ĐOor opens doors at events that are anchored on the Ethereum blockchain.
-//
-//Last update: 11.11.2019
-
 pragma solidity ^0.5.0;
 
 import "@openzeppelin/contracts/ownership/Ownable.sol";
 import "@openzeppelin/contracts/lifecycle/Pausable.sol";
 import "@openzeppelin/contracts/math/SafeMath.sol";
 
-
-    //TODO
-    //1. Review (old) hackathon code created in sleepytime limbo  [work in progress..]
-    //2. Get GSN working
-    //3. Upgradable contracts which includes created user data
-    //4. Create event “tickets” that are transferable (can lead to a market)
-
-
-contract DoorEvent is Ownable, Pausable {
-    //SafeMath library for calculations
+contract Door is Ownable, Pausable {
     using SafeMath for uint;
-
-
-    /**
-     * Variables
-     */
-
-    //Contract essential data
-    string private eventName;
-    uint private ticketPrice;
-    bool private rsvpCanWithdrawFunds;
-
-    //Attendees ("RSVPs") can either be
-    enum rsvpStatus {
-        NONE,       //0: unregistered,
-        REGISTERED, //1: registered (ticket owner) or
-        ATTENDED    //2: can have attended.
+    
+    string public eventName;
+    uint public ticketPrice;
+    bool public allowsWithdraws;
+
+    enum TicketStatus {
+        NONE,       
+        REGISTERED,
+        ATTENDED
     }
-    //Create attendee ("RSVP") struct
-    struct rsvpStruct{
-        rsvpStatus ticketStatus;
+    
+    struct attendeeStruct{
+        TicketStatus ticketStatus;
```

Comment:
since we're talking about "prices" here, I think it's a good thing consistently refer to everyhing as a "ticket".

Reply:
good point, thanks for checking